AsyncDisplayKit/ASTextNode.mm

```diff
@@ -1042,8 +1047,10 @@ - (void)setShadowColor:(CGColorRef)shadowColor
   if (_shadowColor != shadowColor) {
     if (shadowColor != NULL) {
       CGColorRetain(shadowColor);
+      CGColorRelease(_shadowColor);
```

Comment:
Hmmm... I think we need to move this out of the if statement and wrap it in a new one: `if (_shadowColor != NULL)` because no matter what the new value is, if we have an old value we must release it